docs/modular-steps-management.md

```diff
@@ -68,84 +69,95 @@
 ### Examples
 
 Here is an example of the form step's attributes. It presents a form with two sections, and an example of each kind of input type. 
-> NB : this example object will soon be provided in the admin.
+> NB : this example object is provided in the admin, in the onboarding section: 'Form step example'.
 
 ```json
 {
-  "subtype": "onb-adm-form",
+  "subtype": "form-step",
   "form": {
     "section1": {
       "title": "My section title",
       "inputs": {
-        "firstName": {
+        "typeText": {
```

Comment:
I don't like the generic names over the concrete specific ones previously used in the examples.